### ISSUE TITLE
orientation problem of background image in plot_glass_brain 

### ISSUE THREAD

**Issue by chrplr** [nilearn/nilearn#1815]
When using `plot_glass_brain(..., display_mode='lyrz')` on a volume with no non-zero data in the left hemisphere, the brain contour is flipped (i.e. the left hemi. is displayed like a right hemi.) See picture below.

![figure_1](https://user-images.githubusercontent.com/1530674/46949283-2b437580-d081-11e8-9fe3-63122eb44ab9.png)

**Comment by GaelVaroquaux** [comment]
I think that this is an issue in one of the matplotlib versions only. Can you give us your version?

**Comment by KamalakerDadi** [comment]
Here is the reproducible script:

```python
from nilearn import plotting
from nilearn import datasets
from nilearn.image import new_img_like

ho = datasets.fetch_atlas_harvard_oxford('sub-maxprob-thr25-2mm',
                                         symmetric_split=True)
data = ho.maps.get_data()
this_label = (data == 19)
img = new_img_like(ho.maps, this_label)
plotting.plot_glass_brain(img, display_mode='lyrz')
plotting.show()
```

**Comment by Shotgunosine** [comment]
I've got the same issue, reproduces with the above script.
Here's the conda list for my environment:
```
# packages in environment at /Users/nielsond/miniconda3/envs/emily_rev:
#
# Name                    Version                   Build  Channel
appnope                   0.1.0                    py37_0  
attrs                     19.1.0                   py37_1  
backcall                  0.1.0                    py37_0  
blas                      1.0                         mkl  
bleach                    3.1.0                    py37_0  
ca-certificates           2019.5.15                     0  
certifi                   2019.6.16                py37_0  
cycler                    0.10.0                   py37_0  
dbus                      1.13.6               h90a0687_0  
decorator                 4.4.0                    py37_1  
defusedxml                0.6.0                      py_0  
entrypoints               0.3                      py37_0  
expat                     2.2.6                h0a44026_0  
freetype                  2.9.1                hb4e5f40_0  
gettext                   0.19.8.1             h15daf44_3  
glib                      2.56.2               hd9629dc_0  
icu                       58.2                 h4b95b61_1  
intel-openmp              2019.4                      233  
ipykernel                 5.1.1            py37h39e3cac_0  
ipython                   7.6.0            py37h39e3cac_0  
ipython_genutils          0.2.0                    py37_0  
ipywidgets                7.4.2                    py37_0  
jedi                      0.13.3                   py37_0  
jinja2                    2.10.1                   py37_0  
joblib                    0.13.2                   py37_0  
jpeg                      9b                   he5867d9_2  
jsonschema                3.0.1                    py37_0  
jupyter                   1.0.0                    py37_7  
jupyter_client            5.2.4                    py37_0  
jupyter_console           6.0.0                    py37_0  
jupyter_core              4.5.0                      py_0  
kiwisolver                1.1.0            py37h0a44026_0  
libcxx                    4.0.1                hcfea43d_1  
libcxxabi                 4.0.1                hcfea43d_1  
libedit                   3.1.20181209         hb402a30_0  
libffi                    3.2.1                h475c297_4  
libgfortran               3.0.1                h93005f0_2  
libiconv                  1.15                 hdd342a3_7  
libpng                    1.6.37               ha441bb4_0  
libsodium                 1.0.16               h3efe00b_0  
llvm-openmp               4.0.1                hcfea43d_1  
markupsafe                1.1.1            py37h1de35cc_0  
matplotlib                3.1.0            py37h54f8f79_0  
mistune                   0.8.4            py37h1de35cc_0  
mkl                       2019.4                      233  
mkl-service               2.0.2            py37h1de35cc_0  
mkl_fft                   1.0.12           py37h5e564d8_0  
mkl_random                1.0.2            py37h27c97d8_0  
nbconvert                 5.5.0                      py_0  
nbformat                  4.4.0                    py37_0  
ncurses                   6.1                  h0a44026_1  
nibabel                   2.4.1                    pypi_0    pypi
nilearn                   0.5.2                    pypi_0    pypi
notebook                  5.7.8                    py37_0  
numpy                     1.16.4           py37hacdab7b_0  
numpy-base                1.16.4           py37h6575580_0  
openssl                   1.1.1c               h1de35cc_1  
pandas                    0.24.2           py37h0a44026_0  
pandoc                    2.2.3.2                       0  
pandocfilters             1.4.2                    py37_1  
parso                     0.5.0                      py_0  
patsy                     0.5.1                    py37_0  
pcre                      8.43                 h0a44026_0  
pexpect                   4.7.0                    py37_0  
pickleshare               0.7.5                    py37_0  
pip                       19.1.1                   py37_0  
prometheus_client         0.7.1                      py_0  
prompt_toolkit            2.0.9                    py37_0  
ptyprocess                0.6.0                    py37_0  
pygments                  2.4.2                      py_0  
pyparsing                 2.4.0                      py_0  
pyqt                      5.9.2            py37h655552a_2  
pyrsistent                0.14.11          py37h1de35cc_0  
python                    3.7.3                h359304d_0  
python-dateutil           2.8.0                    py37_0  
pytz                      2019.1                     py_0  
pyzmq                     18.0.0           py37h0a44026_0  
qt                        5.9.7                h468cd18_1  
qtconsole                 4.5.1                      py_0  
readline                  7.0                  h1de35cc_5  
scikit-learn              0.21.2           py37h27c97d8_0  
scipy                     1.2.1            py37h1410ff5_0  
seaborn                   0.9.0                    py37_0  
send2trash                1.5.0                    py37_0  
setuptools                41.0.1                   py37_0  
sip                       4.19.8           py37h0a44026_0  
six                       1.12.0                   py37_0  
sqlite                    3.28.0               ha441bb4_0  
statsmodels               0.9.0            py37h1d22016_0  
terminado                 0.8.2                    py37_0  
testpath                  0.4.2                    py37_0  
tk                        8.6.8                ha441bb4_0  
tornado                   6.0.3            py37h1de35cc_0  
traitlets                 4.3.2                    py37_0  
wcwidth                   0.1.7                    py37_0  
webencodings              0.5.1                    py37_1  
wheel                     0.33.4                   py37_0  
widgetsnbextension        3.4.2                    py37_0  
xz                        5.2.4                h1de35cc_4  
zeromq                    4.3.1                h0a44026_3  
zlib                      1.2.11               h1de35cc_3
```

**Comment by Shotgunosine** [comment]
For the next person that has this issue, as a workaround, if you can identify the axis in your figure that is oriented incorrectly, you can just flip the x limits. Here's the reproducible example with this workaround added:

```
from nilearn import plotting
from nilearn import datasets
from nilearn.image import new_img_like

fig, ax = plt.subplots(1,1)
ho = datasets.fetch_atlas_harvard_oxford('sub-maxprob-thr25-2mm',
                                         symmetric_split=True)
data = ho.maps.get_data()
this_label = (data == 19)
img = new_img_like(ho.maps, this_label)
plotting.plot_glass_brain(img, display_mode='lzr', axes=ax)
axi = fig.get_axes()[-1]
xmin,xmax = axi.get_xlim()
axi.set_xlim((xmax,xmin))
plotting.show()
```

**Comment by KamalakerDadi** [comment]
PR #1888 is already there to fix this. @Shotgunosine do you have time to review as you know the crux to fix this ? Can you try my PR with your reproducible code and help us know if it is fixed ?

Thanks in advance @Shotgunosine

**Comment by kchawla-pi** [comment]
Fixed by PR #1888
